assets/css/buttons.css

```diff
@@ -14,13 +14,13 @@
   font-weight: 400;
   font-style: normal;
   line-height: 1;
-  font-size: .75rem;
+  font-size: .75em;
```

Comment:
What's the justification for using em? Particularly with font-size. I didn't catch it before but why not just px? Setting the font-size leads to ambiguity in what your font size truly is.